front-end/myntra/src/index.js

```diff
@@ -4,8 +4,11 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import '@fortawesome/fontawesome-free/css/all.min.css';
+
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
+<script src="https://kit.fontawesome.com/a5f0b11436.js" crossorigin="anonymous"></script>
```

Comment:
why use this script tag in this file? can we use it index.html ?

Reply:
Comment fixed, please initiate merge!